Cargo.toml

```diff
@@ -25,7 +25,7 @@ udigest = { version = "0.2.1", default-features = false }
 digest = { version = "0.10", default-features = false }
 sha2 = { version = "0.10", default-features = false }
 
-rand = "0.8"
+rand = { version = "0.8", default-features = false }
```

Comment:
Why? This enables default features in all subcrates that depend on `rand`. We need default features in all of them?

Reply:
Maybe better specify the exact features you need?

Reply:
Are you reading this backwards? I disabled default features in rand; because now rand is used in key-share, but by default rand requires `std`. So I disable default features to disable it

Reply:
Yes I was reading it backwards 🤦‍♂️